# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/browser.js
# Commit: 4b76024db21c
# Full Hash: 4b76024db21cff006a7424a35c412c6e0a153eb0
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-09-22 03:47:42
# Regressor Bug: 1655866
# File Overlap Count: 1
# Description:
#   Bug 1655866: Part 5 - Use native PermitUnload implementation from front-end code. r=nika,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D88318
# ==============================================================================

diff -r 55fb069319f1 -r 4b76024db21c browser/base/content/browser.js
--- a/browser/base/content/browser.js	Mon Sep 21 22:41:01 2020 +0000
+++ b/browser/base/content/browser.js	Mon Sep 21 22:41:04 2020 +0000
@@ -49,7 +49,6 @@
   PageThumbs: "resource://gre/modules/PageThumbs.jsm",
   PanelMultiView: "resource:///modules/PanelMultiView.jsm",
   PanelView: "resource:///modules/PanelMultiView.jsm",
-  PermitUnloader: "resource://gre/actors/BrowserElementParent.jsm",
   PictureInPicture: "resource://gre/modules/PictureInPicture.jsm",
   PlacesUtils: "resource://gre/modules/PlacesUtils.jsm",
   PlacesUIUtils: "resource:///modules/PlacesUIUtils.jsm",